src/menu_events.cpp

```diff
@@ -454,6 +454,29 @@ void menu_handler::show_enemy_moves(bool ignore_units, int side_num)
 			gui_->highlight_another_reach(path);
 		}
 	}
+	
+	//   Fetch cursor position, in order to later find a possible
+	// unit under it.
+	int x = -1;
+	int y = -1;
+	// auto bitmask =
```

Comment:
Please remove this commented out variable and indent the subsequent `SDL_GetMouseState` call appropriately.

Reply:
Will comply request.